### DISCUSSION TITLE
Difference with Shoulder Surfing Reloaded

### DISCUSSION THREAD

**Discussion by MrCode200** [Leawind/Third-Person#160] [General]
Hey :), I hope it's ok to ask here, I couldn't find anywhere else to ask (nod discord server or anything else)😅. I was wondering what the difference is between your mod and shoulder surfing reloaded as they seem like the same.  
Btw thank you for creating this mod 🙏.

**Comment by Leawind** [comment]
Hello,

If you are unsure about the differences, I would suggest you use Shoulder Surfing Reload, as it seems to have better compatibility and supports more Minecraft versions. I haven't been paying attention to its updates for a while, so I'm not very clear on the features of its latest version.
